modules/brainbrowser/ajax/minc.php

```diff
@@ -30,7 +30,9 @@
 $header      = $_REQUEST['minc_headers'];
 $header_data = $_REQUEST['raw_data'];
 if ($header_data) {
-    passthru("minctoraw -double -normalize $minc_file");
+    $fp = fopen($minc_file, 'r');
+    fpassthru($fp);
```

Comment:
The `fpassthru` documentation recommends using `readfile` in situations like this:

```
If you just want to dump the contents of a file to the output buffer, without first modifying it or seeking to a particular offset, you may want to use the readfile(), which saves you the fopen() call.
```

http://php.net/manual/en/function.fpassthru.php